modules/addbook.js

```diff
@@ -0,0 +1,80 @@
+/* eslint-disable array-callback-return */
+/* eslint-disable consistent-return */
+/* eslint-disable no-unused-vars */
```

Comment:
- [ ] `Linters` are tools that perform static analysis on software in order to recognize and report adherence/non-adherence to some set of coding best practices. Instead of disabling `eslint`, it will be nice to fix the issue.

Reply:
HEY @Mihndim2020, I totally agree with you. However, as you may see in the project requirements it's mentioned that we can disable some rules like the one I did if we are using many js files.
here's the [link](https://github.com/microverseinc/curriculum-javascript/blob/main/books/m1_plain_js_objects_v1_1.md) to the project requirement as you may read. 
Here's the mention.

## TIP: 
If you separated the code in multiple files and you have the following issues with the linter: complaining about having multiple classes at the same file, or unused variables, disable the linter check for that rule in the file with a /* eslint-disable rule-you-want-to-disable */ .